### DISCUSSION TITLE
Is it possible to deploy Nextron app on vercel as regular Nextjs app?

### DISCUSSION THREAD

**Discussion by btvoidx** [saltyshiomix/nextron#150] [Q&A]
I want my app to be available both in browser and on desktop, and I'm not sure whether Nextron is the tool I want to use.

I have no experience with both Nextjs and Electron, but I want to kill to birds with one stone and make a monstrosity of a codebase. :^)

**Comment by Makisuo** [comment]
Any updates to this?

**Reply by btvoidx** [reply]
I played with it at the time, and it appears that it's very easy to do if all pages can be statically exported. I wasn't able to find any way to use SSR or ISR, because these aren't supported by Nextron.

**Reply by sustainjane98** [reply]
Maybe a solution would be to have a monorepo and share all files in renderer between both applications.
